plottr/utils/num.py

```diff
@@ -380,7 +384,8 @@ def crop2d(x: np.ndarray, y: np.ndarray, *arr: np.ndarray) \
     :return: all arrays (incl. x and y), cropped.
     """
     xs, ys = joint_crop2d_rows_cols(x, y)
-    allarrs = [] + [x, y] + [a for a in arr]
+    empty_list: List[np.ndarray] = []
+    allarrs: List[np.ndarray] = empty_list + [x, y] + [a for a in arr]
```

Comment:
doesn't this work?
```suggestion
    allarrs = [x, y] + [a for a in arr]
```

Reply:
I would say so but I always try to not make any code changes while adding types but this does look pointless